### DISCUSSION TITLE
[Idea] Option to do not use colors (show all text black) + Note about Contrast

### DISCUSSION THREAD

**Discussion by The-Gig** [torhus/monster-browser#66] [Ideas]
I was thinking... some servers and players do have quite "rainbowy" colored names, which sometimes are a bit harsh on the eyes.
What about an option to ignore the colors and show all that text as simple black?
![immagine](https://user-images.githubusercontent.com/23258514/204725029-472668e9-58c2-4acc-8781-664e2f915506.png)
By the way, speaking about text colors, the name of the currently selected server/player, when you click on one, is highlighted with a dark blue which makes the black text not very readable. Maybe a lighter shade of blue, to get better contrast? Or making that text of a light grey instead of black?

**Comment by torhus** [comment]
You can try turning the colors off and see it you prefer that, it's `coloredNames` in settings.ini.

The dark blue highlighting is annoying, I can see if there is a way to fix it.

**Reply by The-Gig** [reply]
Colorednames=false in settings.ini does the trick, wonderful! Can I ask you to add a "Colors" checkbox for it inside the Settings window?

By the way, for some reason disabling text color also disables that dark blue highlighting:
![immagine](https://user-images.githubusercontent.com/23258514/205001300-023e6bff-9c83-4fb1-a36a-a6d2b0644e35.png)
... and also changes the "lock" icon in the PW column into a simple "X"...

**Comment by torhus** [comment]
Add issues #67 and #68 for this.
